back-end/service.py

```diff
@@ -105,7 +106,8 @@ def writeSecret(key, repo, user):
         return True
     else:
         log('critical', res.json()['message'])
-        raise ServerError('Secret could not be installed in `{}`'.format(repo), 410)
+        raise ServerError('Secret could not be installed in `{}`'.format(repo),
+            HTTPStatus(417))
```

Comment:
We are not dealing with the `Expect request-header field`, so just return `500`.

Reply:
Changed and pushed.